lib/data_update_scripts/20210712054300_remove_unused_profile_fields.rb

```diff
@@ -3,7 +3,6 @@ class RemoveUnusedProfileFields
     OBSOLETE_FIELDS = %w[
       behance_url
       brand_color1
-      brand_color2
```

Comment:
Our DUS script plays a similar role to the migration file. I'm not sure if it matters or not but perhaps don't remove it yet

Reply:
This makes sense. This change has been reverted.